ote/src/clj/ote/util/clojure.clj

```diff
@@ -0,0 +1,4 @@
+(ns ote.util.clojure)
+
+(defn remove-coll-key-ns [c]
+  (into {} (map (fn [[k v]] [(keyword (name k)) v]) c)))
```

Comment:
No need to change now, but next time this would be easier to understand and change
(into {}
  (map 
    (fn [[key value]] 
     [(keyword (name key)) value]) 
   c)))

Reply:
True :)